config/serialization/CustomerStatus.yaml

```
@@ -0,0 +1,6 @@
+App\Customer\Domain\Entity\CustomerStatus:
+  attributes:
+    id:
+      groups: ['read:customerStatus']
+    value:
+      groups: ['read:customerStatus', 'write:customerStatus']
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)